src/components/Auth/ErrorPage.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import { useQuery } from '../../shared/helper_functions';
+import { useQuery } from '../../shared/utilities';
 import { HttpError } from '../../shared/types/models';
 
 export const ErrorPage = () => {
-    const query = useQuery();
+  const query = useQuery();
 
-    console.log(query.get('message'), query.get('status'));
-    return (
-        <>
-            You on a wrong page son{query.get('status')}: {query.get('message')}
-        </>
-    );
+  console.log(query.get('message'), query.get('status'));
+  return (
+    <>
+      You on a wrong page son{query.get('status')}: {query.get('message')}
+    </>
```

Comment:
I'd remove/comment out the [console log](https://memegenerator.net/img/instances/68620658/a-wild-consolelog-appeared.jpg) and add a space after son:
```suggestion
  // console.log(query.get('message'), query.get('status'));
  return (
    <>
      You on a wrong page son {query.get('status')}: {query.get('message')}
    </>
```